model/src/pyrenew/model/hospitalizations.py

```diff
@@ -86,6 +129,30 @@ def sample_hospitalizations_latent(
         infections: ArrayLike,
         **kwargs,
     ) -> tuple:
+        """Sample number of hospitalizations
+
+        Parameters
+        ----------
+        infections : ArrayLike
+            The predicted infections array.
+        **kwargs : dict, optional
+            Additional keyword arguments passed through to internal
+            sample_hospitalizations_latent calls, should there be any.
```

Comment:
The pass-through here is `sample()` calls generally, not specifically to calls to functions with the same name as this one.
```suggestion
            sample() calls, should there be any.
```